helm/aws-operator/templates/deployment.yaml

```diff
@@ -9,6 +9,10 @@ metadata:
 spec:
   replicas: 1
   revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      app: {{ .Values.project.name }}
+      version: {{ .Values.project.version }
```

Comment:
Good point to have version included. I've been doing these kind of chances in other repos/projects, as preparation for k8s 1.16 where extesnsion/v1beta1 is dropped - have to review them again myself, but please check as well when reviewing dep upgrade PRs, if version or more is needed in selector there too.

Reply:
@sslavic Will do I noticed this when debugging deployment issues on `ginger`. For VOO operators I think we do need both labels.